src/routes/reference/pitch.svx

```diff
@@ -8,4 +8,44 @@ flair: reference
 category: Analyse Sound
 ---
 
+<script>
+  import ResourceLink from '$lib/components/ResourceLink.svelte';
+  import Image from '$lib/components/Image.svelte';
+  import Audio from '$lib/components/Audio.svelte';
+</script>
 
+
+FluCoMa's Pitch object returns both pitch and confidence values. When no pitch can be detected, a pitch of 0 Hz is returned (or -999.0 when the unit is in MIDI note mode).
```

Comment:
Would prefer something agnostic like The Pitch algorithm in the FluCoMa toolkit

Reply:
👍 but instead of "algorithm" i'm going to say "object" since it is three algorithms.